From c339fde06a13a054cb67b21a2183e9ad5af3a7be Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Fri, 26 Jul 2013 14:28:51 +0800
Subject: [PATCH 461/635] ENGR00272760 fix gralloc wrapper build fail

gralloc wrapper build fail caused by struct change on JB4.3

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 mx6/libgralloc_wrapper/gralloc.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mx6/libgralloc_wrapper/gralloc.cpp b/mx6/libgralloc_wrapper/gralloc.cpp
index bb96a42..4369af3 100755
--- a/mx6/libgralloc_wrapper/gralloc.cpp
+++ b/mx6/libgralloc_wrapper/gralloc.cpp
@@ -82,6 +82,7 @@ struct private_module_t HAL_MODULE_INFO_SYM = {
         lock: gralloc_lock,
         unlock: gralloc_unlock,
         perform: 0,
+        lock_ycbcr: 0,
         reserved_proc: {0}
     },
     framebuffer: 0,
-- 
1.8.0

